.github/workflows/ci.yml

```
@@ -0,0 +1,66 @@
+name: DotNet
+
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
```

Use default branch macro.

```suggestion
      - $default-branch
  push:
    branches:
      - $default-branch
```